eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/UtxoSpec.hs

```diff
@@ -81,7 +81,9 @@ spec = describe "UTxO" $ do
       minFeeDiff <- conwayDiffMinFee tx
       -- check that the difference is the sum of the sizes of the passed reference scripts
       minFeeDiff
-        `shouldBe` Foldable.foldMap (\s -> scriptSize s <×> refScriptFee) refScriptInToScripts
+        `shouldBe` Foldable.foldMap
+          (\s -> Coin $ floor $ (fromIntegral @Int @Rational $ scriptSize s) * unboundRational refScriptFee)
+          refScriptInToScripts
```

Comment:
That formula is incorrect. You first need to sum up all the sizes and only then multiply the result bu the fee, because if you do multiplication and rounding on per script basis, then you can accumulate loss with each script, which will make this invalid.

It worked before, since there was no rounding.

Reply:
Ah! :bow: Thank you!